__std__/nix/plutus/library/plutus-project.nix

```diff
@@ -102,6 +102,8 @@ let
           # > Error: C stack usage  17556409549320 is too close to the limit
           # > Fatal error: unable to initialize the JI
           plutus-core.components.exes.generate-cost-model.buildable = lib.mkForce false;
+
+          plutus-core.components.benchmarks.cost-model-test.buildable = lib.mkForce false;
```

Comment:
This is the only job left that fails. 
It fails with the same error as `generate-cost-model` above (stack use too close to the limit)